simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/StationSimulatorFunctionalTest.java

```diff
@@ -637,6 +636,43 @@ void shouldGetHeartbeatIntervalWithGetVariablesRequest() {
         });
     }
 
+    @Test
+    void shouldSendNotifyReportOnGetBaseReportRequest() {
+        mockSuccessfulGetBaseReportAnswer();
+
+        stationSimulatorRunner.run();
+
+        GetBaseReportRequest request = new GetBaseReportRequest().withRequestId(200).withReportBase(FULL_INVENTORY);
+
+        String payload = new Call(UUID.randomUUID().toString(), ActionType.GET_BASE_REPORT, request).toJson();
+
+        await().untilAsserted(() -> {
+            List<Call> stationCalls = server.getReceivedCalls(STATION_ID);
+            assertThat(stationCalls.stream()).anyMatch(call -> call.getActionType() == ActionType.BOOT_NOTIFICATION);
+        });
+
+        stationSimulatorRunner.getStation(STATION_ID).sendMessage(new StationMessage(STATION_ID, StationMessage.Type.OCPP_MESSAGE, payload));
+
+        await().untilAsserted(() -> {
+            List<Call> stationCalls = server.getReceivedCalls(STATION_ID);
+
+            List<NotifyReportRequest> notifyReportRequests =
+                    stationCalls
+                            .stream()
+                            .filter(call -> call.getActionType() == ActionType.NOTIFY_REPORT)
+                            .map(call -> NotifyReportRequest.class.cast(call.getPayload()))
+                            .collect(toList());
+
+            sort(notifyReportRequests, new NotifyReportComparator());
```

Comment:
Since implementation does not have shuffling I think sorting can be omitted

Reply:
done (will push changes and notify you to take a look)